### PR TITLE
Colinanderson/add raster from file

### PR DESCRIPTION
**Pull request by colinanderson**
## Links and Data
Sample Epic: https://devtopia.esri.com/runtime/kotlin/issues/1134
- a [vTest](https://runtime-kotlin.esri.com/view/all/job/vtest/job/sampleviewer/) Job for this PR has been run
  - [x] link: https://runtime-kotlin.esri.com/view/all/job/vtest/job/sampleviewer/59

## What To Review
-  Review the code to make sure it is easy to follow like other samples on Android
- `README.md` and `README.metadata.json` files

## How to Test
Run the sample on the sample viewer or the repo.

### CODE REVIEW

**Review by alan-edi — APPROVED** [review]
@colinanderson Looks good. Just one tiny suggestion.

File: samples/add-raster-from-file/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+## Additional information
+
+See the topic [What is raster data?](http://desktop.arcgis.com/en/arcmap/10.3/manage-data/raster-and-images/what-is-raster-data.htm) in the *ArcMap* documentation for more information about raster images.
+
+This sample uses the GeoViewCompose Toolkit module to be able to implement a Composable MapView.
+
+## Tags
+
+data, image, import, layer, raster, visualization
```
Comment (alan-edi) [inline samples/add-raster-from-file/README.md:34]: Oh, one other thing, [the epic](https://devtopia.esri.com/runtime/kotlin/issues/4961) says to:
> Add the tags: `geoviewcompose`, `toolkit` to readme and metadata.json file

File: samples/add-raster-from-file/src/main/java/com/esri/arcgismaps/sample/addrasterfromfile/components/AddRasterFromFileViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+    // create a raster
+    val raster = Raster.createWithPath(provisionPath +
+            File.separator + "raster-file" + File.separator + "Shasta.tif")
+
+    // create a raster layer
+    val rasterLayer = RasterLayer(raster)
+
+    val arcGISMap = ArcGISMap(BasemapStyle.ArcGISImagery).apply {
+        operationalLayers.add(rasterLayer) }
```
[the remark this answers is not in the document]
Comment (colinanderson) [reply samples/add-raster-from-file/src/main/java/com/esri/arcgismaps/sample/addrasterfromfile/components/AddRasterFromFileViewModel.kt]: Done

File: samples/add-raster-from-file/README.md (hunk 1 above)
Comment (colinanderson) [reply samples/add-raster-from-file/README.md]: Done

**Review by hudsonmiears — APPROVED** [review]
LGTM 👍

**Pull request merged**